src/transformers/inspection.py

```diff
@@ -0,0 +1,29 @@
+from models import Inspection, InspectionTime, InspectionConfirmation
+
+
+class InspectionTransformer:
+    @staticmethod
```

Comment:
może mapper? albo converter? choć wole to pierwsze :)

Reply:
Jest mapper